### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by LinkMaster111** [KevinNitroG/Public-Stuff#2] [Ideas]
The repository is great!
It would be awesome if it's available in English too.
Also some stuff needs to be sorted into groups like indexes that serve only Vietnamese content and others that could be global

**Comment by KevinNitroG** [comment]
> The repository is great! It would be awesome if it's available in English too. Also some stuff needs to be sorted into groups like indexes that serve only Vietnamese content and others that could be global

I don't intend to make it really publicly or be known widely ((: so... You can use translator to see what it means. And don't share it on any forum, youtube, other social media! Thank you ((: Have a great dayy
